librz/asm/arch/hexagon/hexagon_arch.h

```diff
@@ -40,27 +40,40 @@ typedef struct {
 	RzAsmOp *asm_op;
 } HexReversedOpcode;
 
-#define HEX_PKT_UNK           "?   "
-#define HEX_PKT_SINGLE        "[   "
-#define HEX_PKT_SINGLE_UTF8   "[     "
-#define HEX_PKT_FIRST_UTF8    "┌   "
-#define HEX_PKT_MID_UTF8      "│   "
-#define HEX_PKT_LAST_UTF8     "└   "
-#define HEX_PKT_FIRST_SDK     "{   "
-#define HEX_PKT_SDK_PADDING   "   "
-#define HEX_PKT_LAST_SDK      " }"
-#define HEX_PKT_FIRST         "/   "
-#define HEX_PKT_MID           "|   "
-#define HEX_PKT_LAST          "\\   "
+#define HEX_PKT_UNK         "?   "
+#define HEX_PKT_SINGLE      "[   "
+#define HEX_PKT_SINGLE_UTF8 "[   "
+#if __WINDOWS__
+// regex matching doesn't work on Windows with UTF-8 characters
```

Comment:
Not a big fan of this change. @GustavoLCR, could you please take a look? Is this an inherent problem or something that could be fixed?

Reply:
@GustavoLCR I would be deeply grateful. I haven't understood a thing in the regex code.